src/nwb_conversion_tools/tools/spikeinterface/spikeinterface.py

```diff
@@ -1264,15 +1326,23 @@ def write_sorting(
     units_description : str (optional)
         Text description of the sorting table; recommended to included parameters of sorting method, curation, etc.
     """
+
     assert save_path is None or nwbfile is None, "Either pass a save_path location, or nwbfile object, but not both!"
     if nwbfile is not None:
         assert isinstance(nwbfile, pynwb.NWBFile), "'nwbfile' should be a pynwb.NWBFile object!"
     assert write_as in [
         "units",
         "processing",
     ], f"Argument write_as ({write_as}) should be one of 'units' or 'processing'!"
+
+    write_units_table_in_processing_module = True
+    units_table_name = units_name
+    units_table_description = units_description
+
     if write_as == "units":
         assert units_name == "units", "When writing to the nwbfile.units table, the name of the table must be 'units'!"
+        write_units_table_in_processing_module = False
```

Comment:
You can do the setting of the boolean mediator in one line, to reduce 4 lines and a break to 1 line, no break

```python
   write_units_table_in_processing_module = False if write_as == "units" else True
```

Reply:
Good!